build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -136,14 +136,14 @@ steps:
     continueOnError: true
     condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
 
-  # - script: |
-  #     set -e
-  #     node ./node_modules/playwright/install.js
-  #     VSCODE_REMOTE_SERVER_PATH="$(agent.builddirectory)/azuredatastudio-reh-web-darwin" \
-  #     yarn smoketest --web --headless  --screenshots "$(build.artifactstagingdirectory)/smokeshots"
-  #   displayName: Run smoke tests (Browser)
-  #   continueOnError: true
-  #   condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
+  - script: |
```

Comment:
we don't build web layout during darwin build, as far as I know.  I'm not sure why the CI checks are clean, but the mac build step appears to have some issues https://github.com/microsoft/azuredatastudio/pull/17014/checks?check_run_id=3512329092.